pages/index.vue

```diff
@@ -12,6 +12,17 @@
         <t-i18n>{{ $t('注釈') }}</t-i18n>
       </div>
     </div>
+    <static-info
+      class="mb-4"
+      :url="
+        'https://www.pref.miyazaki.lg.jp/kansensho-taisaku/kenko/hoken/kinkyujitaisengen_covid19.html'
+      "
+      :text="
+        $t(
+          '「緊急事態宣言」が全国一律で5月末まで延長され、また国の基本的対処方針が改定されました'
```

Comment:
こちらですが、緊急事態宣言解除に伴いリンク先の内容が変更されていたので、文章を変えた方が良いと思います。
下は例です。

```suggestion
          '第2波第3波に備えて「新しい生活様式」の実践へ！'
```

Reply:
すみません！
例の通りで良いと思います！